pysolarfocus/components/heating_circuit.py

```diff
@@ -0,0 +1,32 @@
+from .base.component import Component
+from .base.enums import DataTypes,RegisterTypes
+from .base.data_value import DataValue
+
+class HeatingCircuit(Component):
+    def __init__(self,input_address=1100,holding_address=32600) -> None:
+        super().__init__(input_address=input_address,holding_address=holding_address)
+        self.supply_temperature = DataValue(address=0,multiplier=0.1)
+        self.room_temperature = DataValue(address=1,multiplier=0.1)
+        self.humidity = DataValue(address=2,multiplier=0.1)
+        self.limit_temperature = DataValue(address=3,type=DataTypes.UINT)
+        self.circulator_pump = DataValue(address=4,type=DataTypes.UINT)
+        self.mixer_valve = DataValue(address=5,type=DataTypes.UINT)
+        self.state = DataValue(address=6,type=DataTypes.UINT)
+        
+        self.target_supply_temperature = DataValue(address=0,multiplier=10,register_type=RegisterTypes.Holding)
+        self.cooling = DataValue(address=2,register_type=RegisterTypes.Holding)
```

Comment:
The holding registers of _heating circuit_ are the only not working for me at the moment: 
```
13:46:09 ERROR __init__:453 Modbus read error at address=32600: Exception Response(131, 3, IllegalAddress)
```
Reason seems to be that, there is a gap with `32601` not being a used register, however, we try to read it with the range-request. This leads to the error above. I've checked the old implementation, here we do not use range-requests for holding. Nevertheless, it works for all but _heating circuit_. 

Do you have an idea how to fix this?

Reply:
Hm, thats a bit tricky ...
I would like to keep the range reads, but if the system doesn't support it we could tile the reads into multiple slices.
In your example we would build tiles from 32600 to 32600 (1 Register) and then we skip 32601 and then read the next slice from 32602 to [End]. We then need to reconstruct the "skipped" registers with a null value to keep it consistant with the implementation for the input registers. WDYT?

Reply:
Sounds good to me. 👍

Reply:
Alright i will add this tomorrow